plugins/woocommerce-admin/client/products/fields/attribute-field/add-attribute-modal.tsx

```diff
@@ -83,28 +91,28 @@ export const AddAttributeModal: React.FC< AddAttributeModalProps > = ( {
 				values.attributes.filter( ( val, i ) => i !== index )
 			);
 		} else {
-			setValue( `attributes[${ index }]`, [
-				{ id: undefined, terms: [] },
-			] );
+			setValue( `attributes[${ index }]`, [ null ] );
 		}
 	};
 
 	const focusValueField = ( index: number ) => {
-		const valueInputField: HTMLInputElement | null = document.querySelector(
-			'.woocommerce-add-attribute-modal__table-row-' +
-				index +
-				' .woocommerce-add-attribute-modal__table-attribute-value-column .woocommerce-experimental-select-control__input'
-		);
-		if ( valueInputField ) {
-			setTimeout( () => {
+		setTimeout( () => {
+			const valueInputField: HTMLInputElement | null =
+				document.querySelector(
```

Comment:
I think ideally we could use a ref here instead of dom selectors, but we can save that for a follow-up issue as I know that wasn't really a part of this PR.

Reply:
Yeah currently the SelectControl component doesn't allow us to pass in a `ref`, I could definitely add that though as a separate PR